phoenix-core-server/src/main/java/org/apache/phoenix/hbase/index/IndexRegionObserver.java

```diff
@@ -366,7 +365,8 @@ public int getMaxPendingRowCount() {
   private boolean shouldWALAppend = DEFAULT_PHOENIX_APPEND_METADATA_TO_WAL;
   private boolean isNamespaceEnabled = false;
   private boolean useBloomFilter = false;
-
+  private long lastTimestamp = 0;
```

Comment:
Longs are not atomic in Java.
Shouldn't this be atomic / volatile or be accessed in synchronized blocks everywhere ?

Reply:
It is accessed only in synchronized block and should be accessed only in synchronized block. So there is no need to define it as volatile or atomic, which will give the wrong impression.

Reply:
Thanks, you are right.